definitions/checks/check_hotfix_installed.rb

```diff
@@ -46,7 +46,7 @@ def modified_files(package)
   def installed_packages
     packages = []
     repoquery_cmd = execute!('which repoquery')
-    IO.popen([repoquery_cmd, '-a', '--installed', '--qf', '%{ui_from_repo} %{nvra}']) do |io|
+    IO.popen([repoquery_cmd, '-a', '--installed', '--qf', '%{ui_from_repo} %{name}-%{evr}.%{arch}']) do |io|
       io.each do |line|
         repo, pkg = line.chomp.split
         packages << pkg if /satellite|rhscl/ =~ repo[1..-1].downcase
```

Comment:
why are we using `repo[1..-1]` here? that means that  if the repo *starts* with `satellite` (like the new EL8 repos do), this gets mangled to `atellite` and thus won't match?

Reply:
Not able to understand logic behind [1..1], the original pr does not say anything about same as well. So we can consider this as extra fix.